cltk/corpus/swadesh.py

```diff
@@ -22,6 +22,9 @@
 
+# Author: Issam K. H. Halayqa
+swadesh_hbo = [“ 'anî, ‘ānokî ”, “ 'attāh, ‘att ”, “ hû’ ”, “ ”, “ ”, “ hem ”, “ zœh, zoh, z’ot, hû ”, “ ”, “ poh, hennāh ”, “ sām ”, “ mî ”, “ māh ”, “ 'ê, ‘ayyeh ”, “ matay, kî ”, “ ek ”, “ lo, ’al ”, “ kol ”, “ rabbîm, rab, harbeh ”, “ ”, “ ”, “ aher ”, “ ’æhād, ’ahat ”, “ senayim, setayim ”, “ sālôs, selosāh, selôsāh ”, “ 'arbā’, ‘arbā’ā ”, “ hāmes, hamissāh ”, “ gādôl ”, “ ’ārok ”, “ ”, “ ”, “ ”, “ qātān, sā’îr ”, “ qasîr ”, “ ”, “ daq, raq ”, “ 'issāh ”, “ 'is, gæbær ”, “ 'ādām ”, “ yælæd, mosæh ”, “ 'issāh ”, “ 'is, gæbær ”, “ 'em ”, “ 'āb ”, “ ”, “ dāg ”, “ 'ôp, sippôr ”, “ kælæb ”, “ ”, “ nāhās ”, “ ”, “ 'ês ”, “ ”, “ ”, “ perî, ‘eb ”, “ zæra’ ”, “ ”, “ sôres ”, “ ”, “ ”, “ ”, “ ”, “ 'ôr ”, “ ”, “ dām ”, “ 'æsæm, gæræm ”, “ sāmen ”, “ ”, “ qæræn ”, “ ”, “ ”, “ sê’ār ”, “ rôs ”, “ 'ôzen ”, “ 'ayin ”, “ ”, “ pe ”, “ ”, “ lāsôn ”, “ ”, “ pa’am ”, “ sôq, regel ”, “ bæræk ”, “ yād ”, “ ”, “ bæhæn, homæs ”, “ ”, “ ”, “ gab ”, “ ”, “ leb ”, “ ”, “ sātāh ”, “ 'ākal ”, “ ”, “ ”, “ ”, “ ”, “ ”, “ ”, “ sāhaq ”, “ rā’ā ”, “ sāma’ ”, “ yādā’ ”, “ ”, “ ”, “ pāhad, yāre’ ”, “ yāsen ”, “ hāyāh ”, “ met ”, “ hārag, qātal ”, “ lhm G, N ”, “ sayd ”, “ nkh k, hālam, māhas ”, “ kārat, gāzar ”, “ ”, “ ”, “ ”, “ hāpar, kārāh ”, “ ”, “ 'āp, dā’ah ”, “ ”, “ bā’, ‘ātāh ”, “ sākab ”, “ yāsab ”, “ qām ”, “ sbb G, K ”, “ nāpal ”, “ nātan ”, “ 'āhaz, lāqah ”, “ ”, “ ”, “ rāhas, sātap ”, “ ”, “ māsāk, sālap, nāsae ”, “ ”, “ rāmah ydh D, yārāh ”, “ qāsar, ‘āsar ”, “ ”, “ sāpar, mānāh ”, “ 'āmar, dābar, mll G ”, “ sār, zmr D, nh G, D ”, “ ”, “ ”, “ ”, “ ”, “ ”, “ sæmæs ”, “ yāreah ”, “ kôkāb ”, “ mayim ”, “ gæsæm, mātār ”, “ nāhār ”, “ ”, “ yām ”, “ ”, “ 'æbæn ”, “ ”, “ ”, “ 'æræs ”, “ 'ānān, ‘āb ”, “ ”, “ samayim ”, “ rûah ”, “ sælæg ”, “ ”, “ ”, “ 'ês ”, “ ”, “ sārap, bā’ar, hārar ”, “ dæræk, ‘orah ”, “ har ”, “ 'ādom ”, “ ”, “ ”, “ lābān ”, “ ”, “ laylāh ”, “ yôm ”, “ sānāh ”, “ ”, “ ”, “ ”, “ hādās ”, “ yāsan ”, “ tôb ”, “ ra’, ræsa’ ”, “ ”, “ ”, “ ”, “ ”, “ ”, “ ”, “ ”, “ ”, “ ”, “ ”, “ ”, “ rāhôq ”, “ yāmîn ”, “ semôl ”, “ ”, “ be ”, “ 'im, ’et ”, “ we ”, “ 'im ”, “ Bi-glāl, kî ”, “ sêm ”]
```

Comment:
please run your code  locally before pushing it, `ERROR: Failure: SyntaxError (invalid character in identifier (swadesh.py, line 26))`

Reply:
it ran perfectly on my machine.. i dont understand what the problem is here...

Reply:
```
byte-compiling build/bdist.linux-x86_64/egg/cltk/corpus/__init__.py to __init__.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/cltk/corpus/swadesh.py to swadesh.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/cltk/corpus/multilingual/corpora.py to corpora.cpython-35.pyc
byte-compiling build/bdist.linux-x86_64/egg/cltk/corpus/multilingual/__init__.py to __init__.cpython-35.pyc
```